common/app/com/commercetools/sunrise/common/template/engine/handlebars/HandlebarsHelperUtils.java

```diff
@@ -8,17 +8,19 @@
 import java.util.Locale;
 import java.util.Optional;
 
+import static com.commercetools.sunrise.common.template.engine.handlebars.HandlebarsContextFactory.CMS_PAGE_IN_CONTEXT_KEY;
```

Comment:
I think here is a code smell: https://sourcemaking.com/refactoring/smells/feature-envy